geotrellis-common/src/main/scala/org/openeo/geotrelliscommon/package.scala

```diff
@@ -1,10 +1,26 @@
 package org.openeo
 
-import org.openeo.geotrelliscommon.zcurve.SfCurveZSpaceTimeKeyIndex
 import geotrellis.layer.SpaceTimeKey
 import geotrellis.spark.partition.PartitionerIndex
+import org.apache.spark.rdd.RDD
```

Comment:
Would it also be an option to avoid a dependency on spark? Spark is a fairly heavy thing to depend on, and we only introduce it here for one line. 
Only thing is this logic with the keyindex, that you probably want to have in here. Maybe we can define an indexing function in geotrellis common, that is then used in a module that translates the RDD of spacetimekey into a simple list of indices?